pkg/wrangler/context.go

```diff
@@ -187,6 +215,47 @@ func (w *Context) Start(ctx context.Context) error {
 	return nil
 }
 
+func (w *Context) WithAgent(userAgent string) *Context {
```

Comment:
Is this meant to be called from somewhere?

Reply:
Same as above